sanic/signals.py

```diff
@@ -192,7 +192,7 @@ async def dispatch(
         if inline:
             return await dispatch
 
-        task = asyncio.get_running_loop().create_task(dispatch)
+        task = asyncio.get_running_loop().create_task(dispatch, name="signal")
```

Comment:
Do you think we should add a more descriptive name? Also include the handler `__name__`?